protocol_rest.go

```diff
@@ -140,7 +140,8 @@ func (r restClientProtocol) requestNeedsPrep(op *operation) bool {
 	return len(op.restTarget.vars) != 0 ||
 		len(op.request.URL.Query()) != 0 ||
 		op.restTarget.requestBodyFields != nil ||
-		restHTTPBodyRequest(op)
+		restHTTPBodyRequest(op) ||
+		op.restTarget.method == http.MethodGet
```

Comment:
```suggestion
		op.restTarget.method != http.MethodPost
```
Thinking more, this will occur for any other type of request without a body. Safer would be to ignore any non post.